### Лабораторна робота 3. Вплив цифрового способу життя на ментальне здоров’я: історія, розказана даними 
З розвитком цифрових технологій дедалі більше людей проводять значну частину свого часу перед екранами: під час навчання, роботи та дозвілля. Водночас психологічні дослідження вказують на те, що надмірна або неструктурована взаємодія з цифровими пристроями може бути пов’язана зі зниженням якості сну, підвищенням рівня стресу та зменшенням концентрації.

У цьому контексті особливої актуальності набуває аналіз даних, який дозволяє глибше зрозуміти, як саме різні аспекти використання цифрових технологій можуть впливати на ментальний стан людини.

**У цій лабораторній роботі основна увага приділяється розвідковому аналізу даних з акцентом на data storytelling, тобто вмінню знаходити зміст у даних і доносити його у вигляді логічної та переконливої аналітичної історії. Таким чином, у цій лабораторній роботі ви не просто аналізуєте дані, а створюєте розповідь на основі даних (data storytelling), що дозволяє краще зрозуміти досліджувану проблему.**  

В лабораторній роботі використовується набір даних, що містить результати опитування 400 респондентів:  
Кожен рядок описує одного респондента:  
user_id - унікальний ідентифікатор кожного учасника опитування.  
age - вік респондента (повних років).  
gender - стать респондента (наприклад, чоловіча / жіноча / інше).  
occupation - рід занять (наприклад, студент, працівник офісу, фрилансер тощо).  
work_mode - формат роботи або навчання (онлайн, офлайн, гібридний).  
screen_time_hours - загальна кількість годин, проведених перед екранами протягом доби.  
work_screen_hours - кількість годин, проведених за екранами під час роботи або навчання.  
leisure_screen_hours - кількість годин екранного часу, пов’язаного з відпочинком (соцмережі, фільми, ігри тощо).  
sleep_hours - середня тривалість сну за добу (у годинах).  
sleep_quality_1_5 - самооцінка якості сну за шкалою від 1 (дуже поганий сон) до 5 (дуже хороший сон).  
stress_level_0_10 - рівень стресу за шкалою від 0 (без стресу) до 10 (високий рівень стресу).  
productivity_0_100 - суб’єктивна оцінка власної продуктивності або ефективності (у відсотках від 0 до 100).  
exercise_minutes_per_week - кількість хвилин фізичної активності на тиждень.  
social_hours_per_week - кількість годин соціальної взаємодії на тиждень (спілкування з друзями, сім’єю тощо).  
mental_wellness_index_0_100 - інтегральний показник ментального благополуччя (0 - дуже низький, 100 - дуже високий рівень).

In [1]:
import pandas as pd
df = pd.read_csv('digital.csv')
df.head()

,user_id,age,gender,occupation,work_mode,screen_time_hours,work_screen_hours,leisure_screen_hours,sleep_hours,sleep_quality_1_5,stress_level_0_10,productivity_0_100,exercise_minutes_per_week,social_hours_per_week,mental_wellness_index_0_100,Unnamed: 15
0,U0001,33,Female,Employed,Remote,10.79,5.44,5.35,6.63,1,9.3,44.7,127,0.7,9.3,NaN
1,U0002,28,Female,Employed,In-person,7.40,0.37,7.03,8.05,3,5.7,78.0,74,2.1,56.2,NaN
2,U0003,35,Female,Employed,Hybrid,9.78,1.09,8.69,6.48,1,9.1,51.8,67,8.0,3.6,NaN
3,U0004,42,Male,Employed,Hybrid,11.13,0.56,10.57,6.89,1,10.0,37.0,0,5.7,0.0,NaN
4,U0005,28,Male,Student,Remote,13.22,4.09,9.13,5.79,1,10.0,38.5,143,10.1,0.0,NaN


**Завдання:**    
1. Сформуйте узагальнений портрет респондентів:  
які значення показників ментального стану є найбільш типовими і чи є помітні крайні або нетипові значення.   
чи відрізняється ментальний стан залежно від віку;  
чи існують відмінності між різними гендерними групами;  
Сформулюйте перші спостереження: яким виглядає “середній” користувач?  

2. Дослідіть, як пов’язаний екранний час із показниками ментального стану:  
чи змінюється самопочуття при збільшенні часу перед екранами;  
чи є певні порогові значення, після яких ситуація погіршується або покращується.  
зробіть короткі висновки.  

3. Дослідіть, як режим роботи пов’язаний із ментальним станом:  
порівняйте користувачів із різними форматами роботи (віддалена, офісна, змішана тощо);  
визначте, чи відрізняються їхні показники самопочуття, екранного часу та способу життя.  
чи можна сказати, що певний режим роботи пов’язаний із кращим або гіршим станом?  

4. Дослідіть, як ключові аспекти щоденного способу життя пов’язані з ментальним станом.  

> 4.1. Сон як фактор відновлення  
як тривалість сну пов’язана із самооцінкою ментального стану;  
чи відрізняється самопочуття у людей з різною якістю сну (sleep_quality).  
чи можна сказати, що сон є одним із ключових факторів добробуту?  

>4.2. Фізична активність  
як рівень фізичних вправ пов’язаний із рівнем стресу та продуктивністю;  
чи спостерігається тенденція до покращення самопочуття при більш активному способі життя.  
чи фізична активність асоціюється з кращим ментальним станом?  

>4.3. Соціальна взаємодія  
як кількість годин соціальної взаємодії пов’язана з рівнем стресу;  
чи є відмінності у продуктивності та самопочутті залежно від рівня соціальної активності.  
чи соціальна взаємодія виглядає як підтримуючий фактор?  

>4.4. Самооцінка стану респондентів  
Дослідіть, як суб’єктивні оцінки респондентів пов’язані з їхнім ментальним благополуччям.  
як самооцінка якості сну пов’язана з ментальним станом;  
як рівень стресу відображається на загальному самопочутті;  
як рівень продуктивності співвідноситься з ментальним благополуччям.  
які з цих показників мають найбільш помітний зв’язок із ментальним станом?  


5. Формування рекомендацій  
Уявімо, що на основі отриманих інсайтів вам необхідно сформувати практичні рекомендації для державних програм підтримки добробуту населення.  
визначте, які поведінкові фактори (екранний час, режим роботи, сон, фізична активність, соціальна взаємодія) найбільше пов’язані з ментальним станом;  
опишіть, на які групи населення варто звернути особливу увагу;  
запропонуйте, які саме аспекти способу життя доцільно враховувати при формуванні рекомендацій щодо збереження ментального здоров’я.  

**Рекомендації мають бути:**  
конкретними;  
обґрунтованими результатами проведеного аналізу;  
логічно пов’язаними з виявленими закономірностями.  